lib/cdo/graphics/certificate_image.rb

```diff
@@ -153,6 +153,8 @@ def certificate_template_for(course)
         'MC_Hour_Of_Code_Certificate_Hero.png'
       elsif course == ScriptConstants::MINECRAFT_AQUATIC_NAME
         'MC_Hour_Of_Code_Certificate_Aquatic.png'
+      elsif course == 'mee'
```

Comment:
Add to ScriptConstants?

Reply:
I didn't do that because it's not a script ID within our system, as we are not hosting it.  It did strike me that it would be good to have it as a constant somewhere, but I wasn't sure where.

Reply:
Oh ok. I was making assumptions based on previous MC tutorials.  This one is going to be more like a third party tutorial, but we're customizing the certificate for it? 

Are we going to send finishers of this tutorial to code.org/congrats or studio.code.org/congrats? Or are we still planning to land a consolidation by HOC?
https://github.com/code-dot-org/code-dot-org/blob/ae6347cde64519be13149f29aa79817fede224d0/pegasus/helpers/hoc_helpers.rb#L125

Reply:
We were planning on having them pop a browser out of their native app which would go to https://code.org/api/hour/finish/mee, which would take them to https://studio.code.org/congrats?i=[filename].